packages/wrangler/src/config/environment.ts

```diff
@@ -338,6 +338,27 @@ interface EnvironmentNonInheritable {
 		environment?: string;
 	}[];
 
+	/**
+	 * Specify a compiled capnp schema to use
+	 * Then add a binding per field in the top level message that you will send to logfwdr
+	 *
+	 * NOTE: This field is not automatically inherited from the top level environment,
+	 * and so must be specified in every named environment.
+	 *
+	 * @default `{schema:undefined,bindings:[]}`
+	 * @nonInheritable
+	 */
+	logfwdr: {
+		/** capnp schema filename */
+		schema: string;
```

Comment:
```suggestion
		schema: string | undefined;
```
as specified a couple of lines above here